src/components/Index.vue

```diff
@@ -72,7 +76,8 @@ export default {
     if (!this.searchStore) {
       this._localSearchStore = createFromAlgoliaCredentials(
         this.appId,
-        this.apiKey
+        this.apiKey,
+        { stalledSearchDelay: this.stalledSearchDelay }
```

Comment:
Can we consider not adding another "options" parameter, and introduce a getter instead?
So far, the Store is just proxying as much as possible.
Also, if tomorrow the `stalledSearchDelay` is added to the helper, we won't require the option anymore and it would be an issue to be able to have it in the helper + as an option, because we won't know which one to keep.

Reply:
I see what you mean. However my thinking behind choosing this as an option is that this is a feature in the scope of InstantSearch, not the helper. This might evolve but the way it is implemented right now is the way it is in the other libraries.